tests/node_compat/config.toml

```diff
@@ -551,7 +551,6 @@
 "parallel/test-http-header-read.js" = {}
 "parallel/test-http-header-validators.js" = {}
 "parallel/test-http-hex-write.js" = {}
-"parallel/test-http-highwatermark.js" = {}
```

Comment:
This is a false positive. Add a `throw new Error();` in the request callback body and it's never reached.